stream-chat-android/src/main/java/com/getstream/sdk/chat/view/EndlessScrollListener.kt

```diff
@@ -2,9 +2,7 @@ package com.getstream.sdk.chat.view
 
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
-import io.getstream.chat.android.core.internal.InternalStreamChatApi
 
-@InternalStreamChatApi
```

Comment:
I don't really want to provide a utility like this to our customers as public API, it's not super related to our library. They should use their own paging implementations.

Maybe we could opt into using this where we use it in the sample app? Not sure what other solution there is.

Reply:
As discussed on Slack - we are going to keep it public